### PR TITLE
updating BIDS impact page

### PR DESCRIPTION
**Pull request by iamdamion**
Reordered impact page per meeting notes. 

Expanded on much of the history and section text. 

Draft for review. More info need to be filled in, but looking for review of what is done now. 

- this addresses #486

### CODE REVIEW

File: docs/impact/index.md [hunk 1]
```diff
@@ -144,25 +117,32 @@ github-dependents-info --repo bids-standard/pybids --badgemarkdownfile ./dashboa
...
+by maintainers and the community at large, the BIDS standard has seen significant adoption 
+in the neuroimaging research landscape and continues to expand. A more complete history 
+and discussion about the future of the BIDS landscape can be found in this 
+[recent publication](https://pmc.ncbi.nlm.nih.gov/articles/PMC10516110/).
 
-~??? users visiting the website, and ~??? users exploring the BIDS
-Specification, over the past ??? months.
+The BIDS standard continues to be adopted by a larger share of the community each year, 
+and we have had ~??? users visiting the BIDS website and ~??? users exploring the BIDS
+Specification over the past ??? months.
```
Comment (bendhouseart) [inline docs/impact/index.md:136]: @ericearl did this get lost to time in google analytics?

Comment (iamdamion) [reply docs/impact/index.md:136]: it looks like the info is still being collected, but I went to the dashboard and it looks like I don't have permission to view. I initially tried to pull from there. Can someone look into perms? If we're linking it on the site it should also be open to everyone, right?

File: docs/impact/index.md [hunk 2]
```diff
@@ -1,48 +1,17 @@
-# Impact
+# BIDS Impact
 
 This document intends to convey how the BIDS Steering and Maintainers Group can
 support your proposed BIDS grant. In an effort to express the impact of BIDS in
 the brain imaging community, we have shared several of our traffic metrics.
```
Comment (bendhouseart) [inline docs/impact/index.md:6]: I think that this needs to better introduce the following sections of the page and (please chime in here @ericearl @Remi-Gau @christinerogers) an acknowledgements should be mentioned and linked? to somewhere further down on the page.

File: docs/impact/index.md [hunk 3]
```diff
@@ -215,6 +197,47 @@ As of July 1, 2020, we have 183 people signed up for our
...
+
+If you are in the process of putting together a grant, **please email/message
+the pillar lead** that is most closely associated with your proposed grant so we
+may help support this. Our organization is structured into 3 pillars: standard,
+tools, and collaboration.
+
+The respective leads are:
+
+-   Stefan Appelhoff ([stefan.appelhoff@mailbox.org](mailto:stefan.appelhoff@mailbox.org)),
+-   Chris Markiewicz ([markiewicz@stanford.edu](mailto:markiewicz@stanford.edu))
```
Comment (bendhouseart) [inline docs/impact/index.md:211]: I think we should just switch the contact info to be the bids.maintenance@gmail.com address.

Comment (bendhouseart) [reply docs/impact/index.md:211]: Same for below on line 219

File: docs/impact/index.md (hunk 1 above)
Comment (bendhouseart) [reply docs/impact/index.md:136]: Yeah, I would comment out the dashboard for analytics if it's not going to load. I don't have permissions either so all I see if    bunch of boxes with `No Data Set Access`.

**Review by bendhouseart — COMMENTED** [review]
@ericearl this is more your baby than mine, but what I would like to see is to have it read/be a compilation of BIDS greatest hits: community engagement/support, an increase in data sharing (and reuse-ability) as demonstrated by OpenNeuro, and maybe reframe 'How the BIDS team can help you' as 'How you can (make) an impact to/with BIDS'.

File: docs/impact/index.md (hunk 1 above)
Comment (christinerogers) [reply docs/impact/index.md:136]: who can help, @Remi-Gau ?
What's the best way to get unstuck?

Comment (ericearl) [reply docs/impact/index.md:136]: I checked today and the Google Analytics on the BIDS Spec don't appear to have been collected correctly since the release of BIDS 1.10.0. I propose we strike this question-mark-filled sentence and ask the BIDS Maintainers to come back and fill this in later.

Alternatively, after GSOD ends this year, we can ask the Online Presence Working Group to come up with PRs for incremental Impact page improvements.

**Comment by iamdamion** [comment]
Made majority of the suggested updates. Plan to talk more about the intro at meeting. This could be good to go for an incremental change. @Remi-Gau @bendhouseart

File: docs/impact/index.md [hunk 4]
```diff
@@ -1,48 +1,17 @@
...
-For example, please find our [NIH-R24 Brain Initiative BIDS-Derivatives grant](https://osf.io/c3dgx/).
-
-A listing of the previous grants can be found [here](../collaboration/acknowledgments.md)
-
-## dashboards
+## Measuring BIDS Impact
 
 ### Datasets
 
+Over the past ~7 years, the number of shared datasets available to the public has
```
Comment (Remi-Gau) [inline docs/impact/index.md:11]:
```suggestion
Over the years, the number of shared datasets available to the public has 
```

otherwise we have to update this as time goes by

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
will merge the requested change

File: docs/impact/index.md (hunk 1 above)
Comment (Remi-Gau) [reply docs/impact/index.md]: I would say that fixing google analytics is beyond the scope of this.

If we have some dead or useless links, let's comment out the sections that include them, open an issue and fix later.

File: docs/impact/index.md (hunk 4 above)
Comment (bendhouseart) [inline docs/impact/index.md]:
```suggestion
Over the years, the number of shared datasets available to the public has 
```

Comment (bendhouseart) [inline docs/impact/index.md]:
```suggestion
Over the years, the number of shared datasets available to the public has 
```

**Review by bendhouseart — APPROVED** [review]
@iamdamion I'd say to go ahead and add Remi's suggestion and get this merged.

**Review by iamdamion — COMMENTED** [review]
agree. makes sense to me.

File: docs/impact/index.md [hunk 5]
```diff
@@ -1,48 +1,17 @@
-# Impact
+# BIDS Impact
 
 This document intends to convey how the BIDS Steering and Maintainers Group can
 support your proposed BIDS grant. In an effort to express the impact of BIDS in
 the brain imaging community, we have shared several of our traffic metrics.
```
Comment (christinerogers) [inline docs/impact/index.md:5]:
```suggestion
BIDS document  intends to convey how the BIDS Steering and Maintainers Group can
support your proposed BIDS grant. In an effort to express the impact of BIDS in
the brain imaging community, we have shared several of our traffic metrics.
```
The Brain Imaging Data Structure (BIDS) is an open global community driving the standardization of neuroscience data across a broad and growing range of modalities and health research disciplines. First released in June 2016, it is supported by a worldwide research network and endorsed by organizations like the International Neuroinformatics Coordinating Facility ([INCF](https://www.incf.org/)). 
BIDS encompasses: 
* Over 40 domain-specific and modality-specific technical specifications across scientific and technical domains
* Open software conversion and analytics tools, and global infrastructure for collaborating on emerging standards in neuroscience ([Poldrack, et al, 2024](https://doi.org/10.1162/imag_a_00103))
* ~30,000 annual website visits from a very large community of neuroscience researchers actively consulting the BIDS specifications (April 2023-24). 
* Over 300 BIDS contributors currently supporting and maintaining BIDS community resources and tools
* Open working processes through online collaborative tools including GitHub, GitHub Pages, ReadTheDocs, Google Docs, and OSF 

The BIDS core steering and maintainers groups work with the community to optimize open adherence to FAIR principles (Findable, Accessible, Interoperable, and Reusable, [Wilkinson, et al, 2016](https://www.nature.com/articles/sdata201618)) and to actively lower barriers to adoption.

Comment (christinerogers) [reply docs/impact/index.md]: note there are `read more` placeholder links i started adding, should be completed

**Comment by ericearl** [comment]
Here's my [preliminary review WIP](https://docs.google.com/document/d/1o-m0Vl3eOnVUpXyXRLFc6sOguxb8HcgWx5qQKyac7Lo/edit?usp=sharing).

**Comment by christinerogers** [comment]
@iamdamion you re-requested review in Mattermost but not github so i'm going to click those re-review requests for you. 
Let's take a look team ---

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
In would actually split the page: https://github.com/bids-standard/bids-website/issues/452

For example I would move the [Measuring BIDS Impact](https://bids-website--550.org.readthedocs.build/en/550/impact/index.html#measuring-bids-impact) into a different page 'BIDS in numbers'

**Comment by Remi-Gau** [comment]
remove the all contributors badge for the bids validator

image

File: docs/impact/index.md [hunk 6]
```diff
@@ -1,48 +1,46 @@
...
+actively consulting the BIDS specifications (April 2023-24).
+- Over 300 BIDS contributors currently supporting and maintaining BIDS community 
+resources and tools.
+- Open working processes through online collaborative tools including GitHub, 
+GitHub Pages, ReadTheDocs, Google Docs, and OSF.
+  
+The BIDS core steering and maintainers groups work with the community to optimize open 
+adherence to FAIR principles (Findable, Accessible, Interoperable, and Reusable, 
+[Wilkinson, et al, 2016](https://www.nature.com/articles/sdata201618)) and to actively 
+lower barriers to adoption.
```
Comment (bendhouseart) [inline docs/impact/index.md:31]: BIDS seems pretty impactful having just glanced at this section. Anything nit-picky here can be fixed in smaller PR.

**Comment by Remi-Gau** [comment]
also you can remove the row about the bids starter kit

File: docs/impact/index.md [hunk 7]
```diff
@@ -1,48 +1,46 @@
...
-what our fellow colleagues are planning on doing next. These can be grants that
-directly extend our support into a new domain, grants that help BIDS (for example
-OpenNeuro), or unsuccessful grant applications.
-
-For example, please find our [NIH-R24 Brain Initiative BIDS-Derivatives grant](https://osf.io/c3dgx/).
-
-A listing of the previous grants can be found [here](../collaboration/acknowledgments.md)
-
-## dashboards
+## Measuring BIDS Impact
```
Comment (bendhouseart) [inline docs/impact/index.md:33]: This looks much better on the render than it does here, I think stars could table could be a link/hidden, but that can be saved for another PR.

File: docs/impact/index.md [hunk 8]
```diff
@@ -144,25 +150,32 @@ github-dependents-info --repo bids-standard/pybids --badgemarkdownfile ./dashboa
 | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | pybids     | [![](https://img.shields.io/static/v1?label=Used%20by&message=596&color=informational&logo=slickpic)](https://github.com/bids-standard/pybids/network/dependents) |
 
-## Measuring BIDS Impact
+## BIDS History
```
Comment (bendhouseart) [inline docs/impact/index.md:153]: Yes!

**Review by bendhouseart — COMMENTED** [review]
Happy with this, small fixes can be done post merge!
